### DISCUSSION TITLE
Prisma Hierarchical Data Microsoft SQL Server

### DISCUSSION THREAD

**Discussion by lastdoctor** [prisma/prisma#19563] [Q&A]
I created migration

```
CREATE TABLE Folder
(
  node  hierarchyid PRIMARY KEY CLUSTERED,
  level AS node.GetLevel(),
  id    int UNIQUE  NOT NULL,
  name  varchar(20) NOT NULL,
  title varchar(20) NULL,
);
```
after running migration

the table is created everything is okay
but for example if I added a model in `schema.prisma`
and run `npx prisma migration dev`
prisma generates migration to remove table `Folder`

how make prisma skip `Folder` table on migration?

**Comment by ludralph** [comment]
Hi @lastdoctor 👋 

Unfortunately, it is not possible to ignore a specific table in Prisma migration. However, there is a feature request for this exact behavior as specified in this issue. Do add your use case and leave a 👍  to enable us to prioritize it.

- https://github.com/prisma/prisma/issues/8864

Also, I'm curious to know how you created the migration that got you to this state.

**Reply by nurul3101** [reply]
@lastdoctor Did you had a chance to look at @ludralph's response?

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have the a similar problem 🙏
